src/amo/components/AddonDetail.js

```diff
@@ -198,6 +217,7 @@ export class AddonDetailBase extends React.Component {
           </h2>
           <AddonMeta addon={addon} />
           <InstallButton {...this.props} />
+          <AddonCompatibility maxVersion={maxVersion} minVersion={minVersion} />
```

Comment:
Yeah that's a solid idea. I rushed through bits of this and found the double usage of `isCompatibleWithUserAgent()` a code smell. That idea makes sense to me!